README.rst

```diff
@@ -0,0 +1,48 @@
+===============
+django-zencoder
+===============
+
+.. image:: https://img.shields.io/pypi/v/django-zencoder.svg
+    :target: https://pypi.python.org/pypi/django-zencoder
+
+.. image:: https://img.shields.io/pypi/pyversions/django-zencoder.svg
+    :target: https://pypi.python.org/pypi/django-zencoder
+
+.. image:: https://img.shields.io/pypi/djversions/django-zencoder
+    :alt: PyPI - Django Version
+    :target: https://pypi.python.org/pypi/django-zencoder
+
+
+Simple integration of a Django model with a FileField with zencoder.com
+
+Requirements
+============
+
+Django 2.2+ (obviously) and celery to asynchronously run the encoding tasks.
+
+Usage
+=====
+
```

Comment:
If you call it Settings rather than Usage the rest of this section can be shortened:

* Add `django_zencoder` to `INSTALLED_APPS`
* Add `ZENCODER_API_KEY` and `ZENCODER_NOTIFICATION_SECRET`
* Set the `ZENCODER_THUMBNAIL_INTERVAL`
* Add the desired formats, for example ::
    ZENCODER_FORMATS = [
        {'label': 'H.264 (HD)', 'codec': 'h264'},
        {'label': 'H.264', 'codec': 'h264', 'width': 720, 'height': 404},
        {'label': 'VP9 (HD)', 'codec': 'VP9'},
        {'label': 'VP9', 'codec': 'VP9', 'width': 720, 'height': 404},
    ]

Reply:
It is `common usage` to have a `usage` section in the README, so I left it, but added a line before to simplify the bullet list!